README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
+
+
+## FAQ
+
+#### What are the priorities for the project?
+
+Well... We need a working demo. That is the most important thing for now. As [bopke](https://github.com/bopke) (project leader) [mentioned](https://github.com/FunnyGuilds/FunnySectors/issues/6) we need a working demo to avoid burnout.
+
+#### What technology are you going to start with?
+
+* Project language: [Java](https://www.java.com/en/)
+* Test language: [Kotlin](https://kotlinlang.org/)
+* Database: [MySql](https://www.mysql.com/)
+* Message broker: [Redis](https://redis.io/)
+* Minecraft platform: [Spigot](https://www.spigotmc.org/)
+* Proxy: [Velocity](https://velocitypowered.com/)
+
+All the technologies have been discussed [here](https://github.com/FunnyGuilds/FunnySectors/issues)
+
+### Are you planing on adding support for XYZ?
+
+We are planing to have a good abstraction layer so adding anything is not that hard. If you really care you can create issue or pull request. All contributions are welcome!
+
+### Is there any way to contact you?
+
+Yes! You can join project discord [here](https://discord.gg/CYvyq3u)
+
+
+
+
+## Roadmap
+
+- create project (setup [modules](https://github.com/FunnyGuilds/FunnySectors/issues/9))
+
+- create abstraction layer for message broker, database
+
+- create spigot "one source of truth" (Database as a way of storing players equipment)
+
+- split map into small areas (chunks) and add a way to send player between them
+
+- send player equipment using message broker and recive it on the other end
+
+- add some way to sync map between sectors
+
+- optymalize everythin
+
+- release first demo version
+
+## Logo
+
+Well... We do not have one. You are more then welcome to create one :)
+## License
+
+[AGPL 3.0](https://choosealicense.com/licenses/agpl-3.0/#)
```

Comment:
We don't need to mention it here, `LICENSE` file already does it

Reply:
> We don't need to mention it here, `LICENSE` file already does it

Well... we are targeting a dynamic market where having "original" plugins is probably the most important factor for the server owner. We want people to see the license when they try to setup the plugin for themselves. It is not like the `MIT` license that allows a lot more. We are using quite restrictive license (`AGPL`) so having this mentioned in the readme is important. Since we decided to use it we want developers to follow it.

Reply:
It does not take a lot of space in reamed and makes license file more accessible to the end user. Not everyone is aware of `LICENSE` file in code and on mobile code is hidden. I do think that the cost of putting license notice in readme is well worth it.

Reply:
LICENSE file is recognized by github and type of license is displayed either on phone and desktop github pages
![on phone](https://cdn.discordapp.com/attachments/254623242914889729/991312972670578808/IMG_1356.png)
![on desktop](https://cdn.discordapp.com/attachments/254623242914889729/991313023329390672/unknown.png)